app/code/Magento/Catalog/Controller/Adminhtml/Category/Add.php

```diff
@@ -39,16 +46,16 @@ public function __construct(
     /**
      * Add new category form
      *
-     * @return \Magento\Backend\Model\View\Result\Forward
+     * @return ResultInterface
      */
-    public function execute()
+    public function execute(): ResultInterface
     {
-        $parentId = (int)$this->getRequest()->getParam('parent');
+        $parentId = (int) $this->getRequest()->getParam('parent');
```

Comment:
Revert.

Reply:
> Revert.

Why should we revert it? There were no complains about it from static tests and actually seems ok to me.

Reply:
Because it must not be touched, common formatting for Magento code is without space (not enforced by any test yet though).